apps/darts-modernisation/darts-gateway/test.yaml

```diff
@@ -11,3 +11,5 @@ spec:
       environment:
         RESTART_ME: '001'
         DAR_NOTIFY_ENABLED: true
+        ACTIVE_DIRECTORY_B2C_BASE_URI: https://hmctsstgextid.b2clogin.com
```

Comment:
and again is this intentionally using staging ?

Reply:
yes